lib/tasks/deployment/20241112151914_load_regions_and_local_authority_districts_2024_11.rake

```diff
@@ -0,0 +1,24 @@
+namespace :after_party do
+  desc 'Deployment task: load_regions_and_local_authority_districts_2024_11'
+  task load_regions_and_local_authority_districts_2024_11: :environment do
+    puts "Running deploy task 'load_regions_and_local_authority_districts_2024_11'"
+
+    file_name = File.join( File.expand_path(File.dirname(__FILE__)) , "schools-to-lads-2024-11.csv" )
+    #id,RGN22NM,LAD22CD
+    CSV.foreach( file_name, headers: true ) do |row|
+      school = School.find_by_id(row[0])
+      if school.present?
+        region = row[1].present? ? row[1].parameterize.underscore.to_sym : nil
+        lad = LocalAuthorityArea.find_by_code(row[2])
```

Comment:
Do we need to check if the LocalAuthorityArea is found here, or are we pretty confident it will be?

Reply:
If it's not found then setting the value to nil is the right behaviour, I think. I'm using the same list of codes to do the matching so unlikely to be an issue anyway. If the administrative boundaries change then we'll need to update both the database and the boundaries so I think its fine.

Reply:
Ok, sounds good, thought I better check 👍